src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const list = document.querySelectorAll('.population');
+
+const population = [];
+
+for (let i = 0; i < list.length; i++) {
+  population.push(list[i].innerHTML.replaceAll(',', ''));
+}
+
+const total = population.reduce(function(sum, current) {
+  return sum + +current;
```

Comment:
It's better using explicit `Number('123')` method, as such option with 2 + operators with different functionalities inhibits code readability